custom-recipes/timeseries-forecast-1-train-evaluate/recipe.json

```diff
@@ -387,13 +390,14 @@
             "name": "training_options_separator",
             "label": "Training options",
             "type": "SEPARATOR",
+            "description": "Detailed documentation: https://ts.gluon.ai/api/gluonts/gluonts.trainer.html",
             "visibilityCondition": "['choose_algorithms', 'customize_algorithms'].includes(model.forecasting_style)"
         },
         {
             "name": "context_length",
             "label": "Context length",
             "type": "INT",
-            "description": "If -1, same as forecasting horizon",
+            "description": "If -1, same as forecasting horizon. Number of past values deep learning models see to make predictions. Higher values increase runtime.",
```

Comment:
```suggestion
            "description": "Number of input lags. If -1, same as forecasting horizon. Higher values may improve performance but increase runtime.",
```

Reply:
I think we shouldn't talk about improving performance, context_length is not really a performance/running time tradeoff.
Also if we can make the description fit on one line on the right of the param, it would be great.
<img width="777" alt="Screenshot 2020-12-21 at 18 58 52" src="https://user-images.githubusercontent.com/60091564/102807254-98b86280-43be-11eb-91fd-b5eba8c4cbf0.png">

Reply:
Okay - then we remove the performance part.
And yes good point, all descriptions should fit on one line

Reply:
I'm not sure that 'Number of input lags' is more understandable than "Number of past values deep learning models see to make predictions"

Reply:
I was trying to find a shorter version to fit on a single line. Actually, even "Number of input lags" is not short enough, so the description doesn't fit. I am wondering if we should explain "Context length". After all, this parameter is for experts only, and context length is the official technical term.

Reply:
I am actually wondering if Context length should even be a parameter, because it's actually not a "training options" but a per-model setting.